modules/core/03-connection/types/msgs_test.go

```diff
@@ -216,22 +216,22 @@ func (suite *MsgTestSuite) TestNewMsgConnectionOpenConfirm() {
 	}
 
 	testCases := []struct {
+		name    string
 		msg     *types.MsgConnectionOpenConfirm
 		expPass bool
-		errMsg  string
 	}{
-		{testMsgs[0], false, "invalid connection ID"},
-		{testMsgs[1], false, "empty proofTry"},
-		{testMsgs[2], false, "empty signer"},
-		{testMsgs[3], true, "success"},
+		{"invalid connection ID", testMsgs[0], false},
```

Comment:
Can you please get rid of the `testMsgs` slice and put the messages directly in the `testCases` struct? That's the same pattern followed in the tests above. Thanks!

Reply:
Yes. No problem